src/DisplayWeather.js

```diff
@@ -0,0 +1,15 @@
+// DisplayWeather component
+
+const DisplayWeather = () => {
+   return (
+      <section>
```

Comment:
instead of user chooses what's to display, are we displaying all types of weather?

Reply:
We will display Current Weather at all times. I will include 2 buttons that will route to Hourly Forecast and 10 Day Forecast.